### PR TITLE
algernon 0.84

### PR DESCRIPTION
**Pull request by xyproto**
Updated to 0.84. Mostly a bug fix release. Re-added the `--boltdb` flag when testing.

### CODE REVIEW

File: Library/Formula/algernon.rb [hunk 1]
```diff
@@ -3,24 +3,16 @@
...
+  url "https://github.com/xyproto/algernon/archive/0.84.tar.gz"
+  sha256 "bc6c9899744ae50736dee1cea1e6c3228d89f8ccf436eb22694664b828e927fc"
   head "https://github.com/xyproto/algernon.git"
 
-  bottle do
-    cellar :any
-    sha256 "12ddd4206dba15312852a8347a2111443be58f0528a0bda43a740d6877999144" => :yosemite
-    sha256 "52fd76910fb29f1a266e0e26dbec4661ef1473dfd8eef58d4f0d6e881c6e01c9" => :mavericks
-    sha256 "411ea5a35dd634e2888eb4786fa7e40a198d77b21f3b1ea8669141e20e3f50bd" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/algernon.rb]: Leave this block as-is and we'll update it when we pull, thanks.

Comment (xyproto) [inline Library/Formula/algernon.rb]: Okay, noted! :+1:

**Comment by xyproto** [comment]
It seems like the testing on mountain_lion is broken. All tests pass: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/28581/testReport/

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/23d6f51eb12ad3efc7fb104a74975e096ab3a4a1. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

**Comment by xyproto** [comment]
Thanks yourself for all your work with Homebrew!

**Pull request closed without merging**